src/MSBuild/MSBuild/Microsoft.Build.CommonTypes.xsd

```diff
@@ -1544,6 +1544,11 @@ elementFormDefault="qualified">
     <xs:element name="AnalysisLevel" type="msb:StringPropertyType" substitutionGroup="msb:Property"/>
     <xs:element name="InstallUrl" type="msb:StringPropertyType" substitutionGroup="msb:Property"/>
     <xs:element name="IsCodeSharingProject" type="msb:boolean" substitutionGroup="msb:Property"/>
+    <xs:element name="IsPackable" type="msb:boolean" substitutionGroup="msb:Property"/>
+    <xs:element name="EnableNETAnalyzers" type="msb:boolean" substitutionGroup="msb:Property"/>
+    <xs:element name="EnableDefaultCompileItems" type="msb:boolean" substitutionGroup="msb:Property"/>
+    <xs:element name="EnableDefaultNoneItems" type="msb:boolean" substitutionGroup="msb:Property"/>
+    <xs:element name="EnableDefaultItems" type="msb:boolean" substitutionGroup="msb:Property"/>
```

Comment:
@dsplaisted @KathleenDollard are we willing to "advertise" these? Should we add descriptions?

Reply:
Yes, I think it's fine to advertise these.

Suggested descriptions:

- EnableDefaultCompileItems - Enable default Compile item globs for source files
- EnableDefaultNoneItems - Enable default None item globs (which cover most files in the project not covered by other globs)
- EnableDefaultItems - Defaults to true, and if set to false will disable all default item globs.

@mikadumont for `EnableNetAnalyzers`